bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5430,15 +5460,15 @@ msgstr ""
 #: bika/lims/browser/dashboard/dashboard.py:727
 #: bika/lims/profiles/default/workflows/bika_sample_workflow/definition.xml
 msgid "Sample received"
-msgstr ""
+msgstr "Muestra recepcionada"
 
 #: bika/lims/browser/dashboard/dashboard.py:693
 msgid "Sample reception pending"
-msgstr ""
+msgstr "Muestra pendiente de recepción"
```

Comment:
@xispa same as my previous comment.

 _recibidas_ sounds more natural to me, but maybe the technical word is _recepcionadas_ so I am ok with whatever you prefer.